SolidStateGroup/bullet-train-api
# feat: Add tags for GitHub integration FE

Thanks for submitting a PR! Please check the boxes below:


## Changes

### Enable tagging and labeling:

<img width="864" alt="Screenshot 2024-08-07 at 3 02 32 PM" src="https://github.com/user-attachments/assets/c9277fd2-2018-4656-8eac-fc17d3f5513b">


### Add tags for GitHub integrations in the FE

<img width="237" alt="Screenshot 2024-05-27 at 9 19 10 AM" src="https://github.com/Flagsmith/flagsmith/assets/41410593/08c3d0de-9c31-451a-b0cc-361308f6879a">

<img width="1039" alt="Screenshot 2024-05-27 at 5 50 42 PM" src="https://github.com/Flagsmith/flagsmith/assets/41410593/7da48c0d-dac8-41f3-993d-a02b54aaf997">

### Add label in the Issue/PR

<img width="1329" alt="Screenshot 2024-08-07 at 3 50 24 PM" src="https://github.com/user-attachments/assets/e5421a8e-440f-43d3-a20c-210fdea16180">


## How did you test this code?

- Use the API branch https://github.com/Flagsmith/flagsmith/pull/4028 for testing.
- Create a Gh integration.
- Enable tagging and labeling.
- Link an issue/PR with a feature.
- See the label in the issue/PR, and the tag in the feature.
